resilience4j-circuitbreaker/src/test/java/io/github/resilience4j/circuitbreaker/utils/CircuitBreakerUtilIIsCallPermittedTest.java

```diff
@@ -0,0 +1,56 @@
+package io.github.resilience4j.circuitbreaker.utils;
+
+import io.github.resilience4j.circuitbreaker.CircuitBreaker;
+import io.github.resilience4j.circuitbreaker.CircuitBreaker.State;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.util.Collection;
+
+import static io.github.resilience4j.circuitbreaker.utils.CircuitBreakerUtil.isCallPermitted;
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(Parameterized.class)
+public class CircuitBreakerUtilIIsCallPermittedTest {
```

Comment:
Could you please rename it to `CircuitBreakerUtilTest`

Reply:
It's already taken by another test. junit 4 does not support nested tested classes off shelf. Unfortunately if I'd include test method from `CircuitBreakerUtilTest` it will be executed n times (as per parameters).

Reply:
Ok, I understand.
Can we then fix the typo ? :)
`CircuitBreakerUtilIsCallPermittedTest`

Reply:
fixed